test/unit/strings_spec.lua

```diff
@@ -8,6 +8,42 @@ local to_cstr = helpers.to_cstr
 local strings = cimport('stdlib.h', './src/nvim/strings.h',
                         './src/nvim/memory.h')
 
+describe('vim_strsave_escaped()', function()
+  local vim_strsave_escaped = function(s, chars)
+    local res = strings.vim_strsave_escaped(to_cstr(s), to_cstr(chars))
+    local ret = ffi.string(res)
+
+    -- Explicitly free memory so we are sure it is allocated: if it was not it 
+    -- will crash.
+    strings.xfree(res)
+    return ret
+  end
+
+  it('precedes by a backslash all chars from second argument', function()
+    eq('\\a\\b\\c\\d', vim_strsave_escaped('abcd','abcd'))
+  end)
+
+  it('precedes by a backslash chars only from second argument', function()
+    eq('\\a\\bcd', vim_strsave_escaped('abcd','ab'))
+  end)
+
+  it('cannot wrap already escaped symmbols (\n is already "line feed")', function()
```

Comment:
- `\n` in this description will be a literal newline. use `[[]]` instead of `'` to delimit the string
- typo: symmbols

That being said, I don't actually understand the purpose of this test: it's passing a literal newline and the newline gets a `\` prepended. Since a newline in C is just another character, this test is the same as the ones above.